docs/aurora/data-science/jupyter.md

```diff
@@ -0,0 +1,136 @@
+# Running JupyterLab and Notebook on Aurora
+
+ALCF provides a [JupyterHub](https://jupyter.alcf.anl.gov/) for running JupyterLab and Notebook on Polaris and Sophia. While this service is not yet available on Aurora, you can still run JupyterLab and Notebook using ssh tunneling and port forwarding.
+
+---
+
+## Prerequisites
+Please note that you only need a terminal (to ssh into Aurora) and a browser on your local machine (laptop or desktop). All the required packages need to be installed on Aurora only.
```

Comment:
```suggestion
Please note that you only need a terminal (to SSH into Aurora) and a browser on your local machine (laptop or desktop). All the required packages need to be installed on Aurora only.
```

I would suggest capitalizing SSH when using in a sentence as a verb, and not the command line command `ssh`

Reply:
I debated about it after DrDoc/gpt-4o recommended as well, but felt ssh is more natural and common. Though I am fine with the change.